src/main/java/glowredman/txloader/MinecraftHook.java

```diff
@@ -0,0 +1,13 @@
+package glowredman.txloader;
+
+import java.util.List;
+
+@SuppressWarnings("unused")
+public class MinecraftHook {
+
+    @SuppressWarnings({ "rawtypes", "unchecked" })
+    public static List insertForcePack(List resourcePackList) {
```

Comment:
Pretty sure you can use `List<IResourcePack>` here to avoid the rawtypes/unchecked warnings.